stringscore_test.go

```diff
@@ -41,6 +41,15 @@ func TestScore(t *testing.T) {
 	}
 }
 
+func TestExactMatchIsPrefferedOverFuzzyMatch(t *testing.T) {
+	query := "backend"
+	fuzzyScore := stringscore.Score("vendor/github.com/coreos/go-oidc/key/manager.go", query)
+	exactScore := stringscore.Score("pkg/backend/trace.go", query)
+	if fuzzyScore >= exactScore {
```

Comment:
I am confused; can you explain this? Higher score is better right? Shouldn't "pkg/backend/trace.go" get a higher score than "vendor/github.com/coreos/go-oidc/key/manager.go" for a query of "backend"

Reply:
nevermind